README.md

```diff
@@ -64,6 +64,37 @@ The build process also creates ```.stamps``` and ```.tmp-output```. The first co
 stamp files to keep track of which parts have been built, while the second contains
 intermediate build output (e.g., kernel images and ISO files).
 
+# Building Windows images
+
+You need to first get the ISO file for the Windows version you would like to install.
+You can download these images from MSDN. The hash and the name of the ISO is
+specified in the  ``images.json`` file. Place these files in the ``iso`` folder.
+
+S2E officially supports images listed in ``images.json``. Other images may
+work too but we did not test them. If you want to add support for new images,
+you will need to also update ``s2e.sys`` in the ``guest-tools`` repository in order
+to support the different kernels. Please refer to the documentation in ``guest-tools``.
+
+
+```
+cd /home/user/s2e/guest-images
+
+mkdir iso && cd iso
+# Download Windows ISO images (e.g., from MSDN)
+# See images.json for details.
+wget ...
+cd ..
+
+S2E_INSTALL_ROOT=/home/user/s2e/build/opt \
+  S2E_LINUX_KERNELS_ROOT=/home/user/s2e-linux-kernel \
```

Comment:
Why is `S2E_LINUX_KERNELS_ROOT` specified in the Windows image build? Should this be `ISODIR`?